frontend/lib/src/components/elements/Video/Video.tsx

```diff
@@ -231,12 +231,12 @@ export default function Video({
       }
     >
       {subtitles &&
-        subtitles.map((subtitle: Subtitle, idx: number) => (
+        subtitles.map((subtitle: ISubtitleTrack, idx: number) => (
           <track
             key={idx}
             kind="captions"
-            src={endpoints.buildMediaURL(subtitle.url)}
-            label={subtitle.label}
+            src={endpoints.buildMediaURL(subtitle.url as string)}
```

Comment:
**suggestion:** Could we enforce this to be a string rather than side-stepping the typecheck?

```suggestion
            src={endpoints.buildMediaURL(`${subtitle.url}`)}
```